### PR TITLE
Reverts TE-X Ray Rifle to its first iteration

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Reverts the TE-X Ray Rifle to its first iteration, as per its addition in #9144. Changes are as follows:
- Shots are incendiary again, and no longer inflict Microwaved.
- Penetration reduced from 10 to 5.
- Fire rate decreased from 0.25s to 0.5s.

## Why It's Good For The Game
TE-X Ray Rifle is unbalanced, and the several changes that have been made to it have made it like this. Instead, we return to our roots, when all was fine and well.

## Changelog
:cl: Lewdcifer
balance: TE-X Ray Rifle: standard mode shots are incendiary again, and no longer inflict Microwaved.
balance: TE-X Ray Rifle: standard mode penetration reduced from 10 to 5.
balance: TE-X Ray Rifle: standard mode fire rate decreased from 0.25s to 0.5s.
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
This is uh, a lot of nerfs at once. the microwave change alone is huge dps nerf

**Comment by dopamiin0** [comment]
lewdcifer has come to save us from the sins of... also lewdcifer. we are saved

**Comment by imsxz** [comment]
> This is uh, a lot of nerfs at once. the microwave change alone is huge dps nerf

you must not have been on recently

**Comment by lKiseki** [comment]
> This is uh, a lot of nerfs at once. the microwave change alone is huge dps nerf

That's because I overdid it back when I """compensated""" for the fire inflict removal. I'm only bringing it back down from up there.

**Comment by Lumipharon** [comment]
> > This is uh, a lot of nerfs at once. the microwave change alone is huge dps nerf
> 
> That's because I overdid it back when I """compensated""" for the fire inflict removal. I'm only bringing it back down from up there.

honk.

**Comment by BoltedAirlock** [comment]
Or we could just revert your previous "nerf" and go back to it being a mildly annoying weapon with a niche.

Why did it even need a nerf to begin with? The original was fine as-it-was.

**Comment by GrayRachnid** [comment]
> Max. shots per normal cell nerfed from 50 > 30.
> Each shot on standard mode will now inflict one stack instead of five.
> Cost in requisitions increased from 400 > 500.
> Firing delay increased from 0.25s. to 0.4s.
I think these are fine.

> Can now only be fired while wielded.
Questionable?
    
> No longer accepts: laser sights, B.A.S., shoulder mount, gyroscopic stabilizer, and bipod.
This feels excessive to tack on on top of other nerfs. It won't be as strong now, so why restrict it further? Also, the main issue I've noted is the use of scopes with it, not a bipod or a laser sight. Why even remove the gyro if you're removing its ability to be fired onehanded?

**Comment by GrayRachnid** [comment]
Question:
How many shots do you get per cell/experimental cell/powerpack with the new values?

**Comment by aarrisu** [comment]
In my view, these adjustments appear to be unnecessary and just cause more complaining within the xenos and marines as a whole and should just rolled back to their previous state before your PR's 

but I do acknowledge the need for some refinements, such as introducing its own ammo to buy in REQ as it's free ammo you can get everywhere for free and also adjusting its pricing structure. I believe that adopting these changes would enhance the overall effectiveness and balance of the TE-X.

In summary stop fucking with the te-x when it was fine before your prs microwave stack is hated by the community  fire was fine

**Comment by Pariah919** [comment]
Wait nuking things is my job.

**Comment by lKiseki** [comment]
Made several changes based on some feedback I got, I invite peeps to read the PR again; I am almost sure this should be a much better alternative.

**Comment by GrayRachnid** [comment]
> a set amount of stacks

How many stacks?

> Removed the following attachments from the TE-X: laser sight, mini flamer, B.A.S., shoulder mount, gyroscopic stabilizer, and bipod.
> The TE-X must now be wielded to fire. Wielding delay increased from 0.5s. to 1s.

I still feel like both of these are unwarranted when the other changes 'fix' the gun.

> Price unchanged

400 is still very cheap for a gun you can get ammo for it for free, are you sure you want to keep it that way?

**Comment by lKiseki** [comment]
> How many stacks?

6 stacks.

> I still feel like both of these are unwarranted when the other changes 'fix' the gun.

Fair enough.

> Price unchanged

At risk of lowballing it again (god help me if so), I wanna see how it performs first with these changes. If it's still too egregious, we can do a point cost increase later down the line.

**Comment by aarrisu** [comment]
i strongly suggest to keep the old TE-X sound as it is and just slightly adjust its volume from what you told me the new one is the ''som laser sound'' in my honest and GOOD opinion it doesn't quite match the sprite like the old sound does and a little side note with you making cells only take 40 instead of 50 can you also address the powerpacks making it 200 and recharger batteries?

I suggest also making it's price slightly higher by making it 500 or 600

**Comment by lKiseki** [comment]
> i strongly suggest to keep the old TE-X sound as it is and just slightly adjust its volume from what you told me the new one is the ''som laser sound'' in my honest and GOOD opinion it doesn't quite match the sprite like the old sound does and a little side note with you making cells only take 40 instead of 50 can you also address the powerpacks making it 200 and recharger batteries?
> 
> I suggest also making it's price slightly higher by making it 500 or 600

I'll hold off on any further balance changes for now, since I wanna see how it fares after these changes. This includes the price cost and the ammo issue.

**Comment by imsxz** [comment]
> Or we could just revert your previous "nerf" and go back to it being a mildly annoying weapon with a niche.
> 
> Why did it even need a nerf to begin with? The original was fine as-it-was.

incredibly questionable take

**Comment by Lumipharon** [comment]
I'm confused why we need a notburning status effect. Also don't use volkite sounds, smelly.

**Comment by lKiseki** [comment]
> I'm confused why we need a notburning status effect. Also don't use volkite sounds, smelly.

I believe this is a much better solution compared to microwave or being set on fire instantly. TE-X users have to put in a little bit more effort to get that bonus and it keeps its previous gimmick (to an extent).

**Comment by sa1tman** [comment]
most roundabout nerf of 2023

**Comment by Lumipharon** [comment]
The gun needs revisiting but this'll at least not make it hilarious OP.

**Review by Lumipharon — APPROVED** [review]
